sst/Makefile

```diff
@@ -0,0 +1,44 @@
+CXX := $(shell sst-config --CXX)
+CXXFLAGS := -std=c++17 -D__STDC_FORMAT_MACROS -fPIC -DHAVE_CONFIG_H -O3 -I/home/rahat/asimov/SST/sst-core-install/include
+LDFLAGS := $(shell sst-config --ELEMENT_LDFLAGS)
+
+SIMENG_ROOT = /home/rahat/asimov/SimEng
+SIMENG_INSTALL := /home/rahat/asimov/simeng-sst-bin
```

Comment:
Just to double check, I presume these change with main SimEng CMake? i.e. not hard-coded as (or defaults to) these paths

Reply:
This makefile should not be here. I will remove it.